supply_chain_settings/models/res_config.py

```diff
@@ -0,0 +1,57 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#   OpenERP, Open Source Management Solution
+#   Copyright (C) 2004-2010 Tiny SPRL (<http://tiny.be>). All Rights Reserved
+#   $Id$
+#
+#   This program is free software: you can redistribute it and/or modify
+#   it under the terms of the GNU Affero General Public License as published by
+#   the Free Software Foundation, either version 3 of the License, or
+#   (at your option) any later version.
+#
+#   This program is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU Affero General Public License for more details.
+#
+#   You should have received a copy of the GNU Affero General Public License
+#   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp import models, fields, api, _
+
+
+class purchase_config_settings(models.TransientModel):
+    _inherit = 'purchase.config.settings'
+
+    default_purchase_requisition = fields.Boolean(
+        _("Generate Procurement using Call for Bids"),
```

Comment:
@hugho-ad the attributes inside a fields.x() definition are automactly translated. Please remove the _() calls into the fields definition